### PR TITLE
Branch add people to company

### PR DESCRIPTION
**Pull request by geraldngjx**

### CODE REVIEW

**Comment by geraldngjx** [comment]
Notable Changes:
- Add a list of people to a company
- Add test cases for PersonList
- Update test cases for existing classes

File: src/test/java/connectify/model/person/PersonListTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import static connectify.testutil.TypicalPersons.ALICE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import connectify.testutil.PersonBuilder;
+import connectify.testutil.PersonListBuilder;
+
+public class PersonListTest {
```
Comment (victorpengmx) [inline src/test/java/connectify/model/person/PersonListTest.java:13]: Very comprehensive use of test cases to ensure code coverage

File: src/main/java/connectify/storage/JsonAdaptedCompany.java [hunk 2]
```diff
@@ -78,6 +92,12 @@ public JsonAdaptedCompany(Company source) {
...
     public Company toModelType() throws IllegalValueException {
+        final List<Person> personListCompany = new ArrayList<>();
+        for (JsonAdaptedPerson person : this.personList) {
+            personListCompany.add(person.toModelType());
+        }
+        final PersonList modelPersonList = PersonList.fromList(personListCompany);
+
         if (name == null) {
             throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, "Name"));
         }
```
Comment (victorpengmx) [inline src/main/java/connectify/storage/JsonAdaptedCompany.java:103]: Good to check that the input for entering information for a new company is valid

**Review by victorpengmx — APPROVED** [review]
Good work to allow for adding a list of people to company and adding test cases to ensure code coverage. LGTM!

File: src/test/java/connectify/model/person/PersonListTest.java (hunk 1 above)
Comment (geraldngjx) [reply src/test/java/connectify/model/person/PersonListTest.java:13]: Thanks for the feedback! We will continue updating the test cases if appropriate when we make new changes.

**Comment by geraldngjx** [comment]
Fixes #44

**Pull request merged**